t/t005-mirror-links.t

```diff
@@ -24,27 +24,20 @@ Check that links function as expected.
 
 projfs_start test_projfs_simple source target || exit 1
 
-EXPECT_DIR="$TEST_DIRECTORY/$(basename "$0" .t | sed 's/-.*//')"
-
 test_expect_success 'create links' '
 	echo text > target/file &&
 	ln target/file target/link &&
 	ls -li target &&
 	ln -s file target/symlink
 '
 test_expect_success 'check link stat' '
-	stat target/file >link.file.stat &&
-	stat target/link >link.stat &&
-	grep "Inode: " link.file.stat | awk "{print \$4}" >inode &&
-	grep "Inode: $(cat inode)" link.stat
+	test "$(stat -c %i target/file)" -eq "$(stat -c %i target/link)"
 '
 
 test_expect_success 'check symlink stat' '
-	stat target/symlink >symlink.stat &&
-	readlink target/symlink > symlink.read &&
-	grep "^file$" symlink.read &&
-	grep "symbolic link" symlink.stat &&
-	grep "Access: (0777/lrwxrwxrwx)" symlink.stat
+	test "$(readlink target/symlink)" = file &&
+	test "$(stat -c %F target/symlink)" = "symbolic link" &&
+	test $(stat -c %04a target/symlink) -eq 0777
```

Comment:
👍

Reply:
Thanks for the meme and the thumbs-up!  The to-do list of small stuff is shrinking now ...